### DISCUSSION TITLE
Question from JLPCB on what to do with these areas on the gerber files.

### DISCUSSION THREAD

**Discussion by robotmonsterous** [FundamentalFrequency/.github#31] [MCAD/ECAD]
This is exactly the project I was looking for! Very cool. Ok was trying to submit the gerber files to JLPCB and they came back with a quesiton. I'm not sure what to tell them.

Thanks in advance for your input.

<img width="1123" alt="Screen Shot 2022-06-28 at 7 38 13 PM" src="https://user-images.githubusercontent.com/22498542/176339244-2f7ba4aa-4165-4374-ae99-e27cc5359f4b.png">
.

**Comment by stonepreston** [comment]
just tell them to do it as is. No holes or slots. It should come out fine with a yellow lemon.

**Comment by robotmonsterous** [comment]
Thanks. Just did!

**Comment by n0ughtchill** [comment]
Jumping on this thread sorry if this is the wrong place. I’ve never got a pcb printed. Is it as easy as just uploading the gerbs? Also any chance you printed extra and willing to sell one? Delete this comment if not allowed

**Reply by peteseyeview** [reply]
yes, and no. It's easy enough on lcpcb.com. Just need to upload gerber.zip file. It's cheaper if you do a  group buy because minimum order is 5 boards. I should have extra available soon.

**Reply by robotmonsterous** [reply]
Same here.

**Reply by briwil** [reply]
I did mine through Gerber Labs.com- wish I had done (or known about) group buys because mine cost $30 to get made- but I just uploaded the files provided here to them and they made the board without any questions or issues.

**Comment by robotmonsterous** [comment]
I just uploaded the gerber zip file. But yes the minimum order was 5 so I
would sell you one if that's allowed on the board. Let me get them and make
sure they're to spec and I'll let you know.
